core/src/test/java/org/infinispan/protostream/annotations/impl/types/DocumentationExtractorTest.java

```diff
@@ -41,41 +43,107 @@ public void method1() {
       @ProtoDoc("")
       @ProtoDoc("")
       public String field5;
+
+      @MyCustomAnnotation(name = "field6", someBool = true, someEnum = MyCustomAnnotation.MyEnum.TWO, someLong = -100, someInteger = 100)
+      public String field6;
+
+      @MyCustomAnnotation(name = "method2", someBool = true, someEnum = MyCustomAnnotation.MyEnum.TWO, someLong = -100, someInteger = 100)
+      public void method2() {
+      }
+
+      @ProtoDoc("1")
+      @MyCustomAnnotation(name = "field7", someBool = true, someEnum = MyCustomAnnotation.MyEnum.TWO, someLong = -100, someInteger = 100)
+      @ProtoDoc("1")
+      public String field7;
+
+      @ProtoDoc("1")
+      @MyCustomAnnotation(name = "method3", someBool = true, someEnum = MyCustomAnnotation.MyEnum.TWO, someLong = -100, someInteger = 100)
```

Comment:
probably useful to add tests with `@MyCustomAnnotation` and `@org.infinispan.custom.annotations.MyCustomAnnotation`

Reply:
Not sure what you mean. Anything more than `testMixedCustomMethodAnnotation`?

Reply:
I mean to check the default values
```java
@MyCustomAnnotation
public String field7
```

Reply:
Added a test for default values. Notice that these are expanded anyway in the generated `.proto`

Reply:
nice